backend/internal/usecase/transaction.go

```diff
@@ -145,3 +150,41 @@ func (uc *TransactionUsecase) QueryIncludedTransactions(ctx context.Context, txL
 	}
 	return txs, nil
 }
+
+func (uc *TransactionUsecase) QueryTransactionDetail(ctx context.Context, txHash string) (*QueryTransactionDetailResp, *error.Http) {
+	erpcTX, err := uc.erpcDBQuery.QueryFromTransactionDetail(ctx, txHash)
+
+	if err != nil {
+		log.Err(err).Msg("err encountered while querying erpc DB")
+		err := error.NewHttpError(
+			"error encountered while querying for data",
+			"",
+			http.StatusInternalServerError,
+		)
+		return nil, &err
+	}
+	txHashBytes, err := hex.DecodeString(erpcTX.TxHash)
+	if err != nil {
+		err := error.NewHttpError(
+			"unable to decode tx hash",
+			"valid encryptedTx query parameter is required",
+			http.StatusBadRequest,
```

Comment:
Is this really a bad request. It looks to me that this scenario would be an error by the ERPC.

Reply:
yeah, i think i just copied the error and forgot to change it

Reply:
Isnt it still not a bad request?